tests/test_thread_safe_dict.py

```diff
@@ -28,3 +28,10 @@ def test_basic_operations() -> None:
     del tsd["key1"]
     with pytest.raises(KeyError):
         _ = tsd["key1"]
+
+    # Test context manager for atomic operations
+    tsd["counter"] = 0
```

Comment:
Should we give ourselves the guideline 'no uncommon abbreviations'?

Reply:
Nice! Then please also change `tsd` to `thread_safe_dict` or `unit`

Reply:
fixed!